src/SUMMARY.md

```diff
@@ -14,6 +14,12 @@
       - [Using the Data Catalog](cookbooks/analysis/data_catalog.md)
       - [Using the Glean Dictionary](cookbooks/analysis/glean_dictionary.md)
       - [Using the Probe Dictionary](cookbooks/analysis/probe_dictionary.md)
+    - [Data Quality - Intro to Bigeye](cookbooks/data_quality/intro.md)
+      - [Features](cookbooks/data_quality/features.md)
```

Comment:
I wonder if for consistency instead of `data_quality` we should call this section `monitoring` to match the corresponding bqetl command?